release.sh

```diff
@@ -18,18 +19,18 @@ doit() {
   git pull
   mvn clean deploy -DskipTests -Dgpg.skip=false
   ./dist.sh
-  # TODO upload
+  ssh root@querydsl.com "mkdir /var/www/html/static/querydsl/$VERSION"
+  scp -r target/dist/* root@querydsl.com:/var/www/html/static/querydsl/$VERSION/
```

Comment:
Could you add this command as well?
`cd /var/www/html/static/querydsl && unlink latest && ln -sT $VERSION latest`.